src/main/java/com/siemens/ct/exi/core/io/channel/ByteDecoderChannel.java

```diff
@@ -50,7 +50,7 @@ public InputStream getInputStream() {
 	public int decode() throws IOException {
 		int b = is.read();
 		if (b == -1) {
-			throw new EOFException("Premature EOS found while reading data.");
+			throw new EOFException("Premature EOF found while reading data.");
```

Comment:
I *think* the idea (at that time) was to convey that it _may_ be an [end-of-stream](https://www.allacronyms.com/EOS/End_of_Stream) and not only a file issue.
Anyhow, since the exception is a EOFException I am fine changing it... what do you think?